doc/docs/development/self-service.md

```diff
@@ -0,0 +1,22 @@
+
+# Eclipse self-service
+
+The Eclipse Foundation offers self-service of GitHub resources.
+We are using this self-service to customize Github settings, for example to change branch protection rules or other important settings of the Ankaios project.
+The current GitHub configuration is hosted as code inside a separate repository called [.eclipsefdn](https://github.com/eclipse-ankaios/.eclipsefdn).
+
+The settings are in jsonnet format and can be modified by contributors.
+
+A detailed overview of the self-service please have a look into the [self-service handbook](https://www.eclipse.org/projects/handbook/#resources-github-self-service).
+
+## Process of changing the settings
+
+If a configuration needs to be changed the process is the following:
+
+1. Fork the [.eclipsefdn](https://github.com/eclipse-ankaios/.eclipsefdn) repository.
+2. Do the configuration changes (Use the [Eclipse playground](https://eclipse-ankaios.github.io/.eclipsefdn/playground/) for trying out the available settings).
+3. Open a PR pointing from your fork's branch to the [.eclipsefdn](https://github.com/eclipse-ankaios/.eclipsefdn) repository.
+4. Request review from: Ankaios project committer, eclipsefdn-releng, eclipsefdn-security.
+5. After the changes were approved by the reviewers, merge the changes into the main branch.
+
+After the changes were merged an Eclipse member will apply the new settings by using the [otterdog cli](https://otterdog.readthedocs.io/en/latest/).
```

Comment:
Does this mean that although we have a self service, after merging the PR a manual step is required by Eclipse staff?

Reply:
Yes, this is how it is currently. But they have written that they are working on automating such things into CI / CD, but this needs still a little bit of testing for now.

Reply:
OK. But then we need rephrase it to:

```suggestion
After the changes were merged, a member of Eclipse staff will apply the new settings by using the [otterdog cli](https://otterdog.readthedocs.io/en/latest/).
```

because 'member of Eclipse' are all member companies.